dist/Cloudinary.common.js

```diff
@@ -1877,23 +1783,38 @@ function noop() {}
     },
     imageAttrs: function imageAttrs() {
       var className = {
-        "cld-image": true,
-        "cld-fill": this.responsive === "fill",
-        "cld-fill-width": this.responsive === "width",
-        "cld-fill-height": this.responsive === "height"
+        "cld-image": true
       };
+      var responsiveStyle = {
```

Comment:
do we really need to apply our style here? if so, why?

Reply:
These styles were always here, they are now as inline styles.
They are inlined as there is no reason for letting the user to override them.

The rules are for stretching an image to its parent width/height/size when "responsive" behaviour is enabled. This way the proper size is always maintained (user should set a size of a parent-container, not to the image element)  and if an image size changes and image file with a different size is necessary - it is then loaded.